make kubevirt-up is failing with main branch due to missing nfs config

Issue Description:

We're observing that all our CI Prow jobs are currently failing because make kubevirt-up exits with the following error:

```
+++ echo 'NFS requested but no NFS directory specified (KUBEVIRT_NFS_DIR)'
NFS requested but no NFS directory specified (KUBEVIRT_NFS_DIR)
+++ exit 1
++ params=
make[1]: *** [Makefile:166: cluster-up] Error 1
make[1]: Leaving directory '/root/go/src/ap/common-instancetypes/_kubevirt'
```
This issue was introduced by the following commit:
🔗 https://github.com/kubevirt/kubevirt/commit/c91f6dc314fde822acb16e8be156acb34c73dffb

As of the above commit, KUBEVIRT_DEPLOY_NFS_CSI defaults to true.

If KUBEVIRT_NFS_DIR is not defined in the environment, cluster startup fails with the error shown above.


**Proposed Solution:**

Set the following environment variable during CI setup to bypass the issue:
```
export KUBEVIRT_DEPLOY_NFS_CSI=false
```
This will disable the NFS CSI deployment and allow make kubevirt-up to succeed in environments where NFS is not explicitly configured.

--- a/scripts/kubevirt.sh
+++ b/scripts/kubevirt.sh
@@ -16,9 +16,6 @@
 
 set -ex
 
-# Required by the VMPersistentState feature that currently needs a RWX FS mode SC
-# https://kubevirt.io/user-guide/virtual_machines/persistent_tpm_and_uefi_state/
-export KUBEVIRT_DEPLOY_NFS_CSI="${KUBEVIRT_DEPLOY_NFS_CSI:-true}"
 export KUBEVIRT_MEMORY_SIZE="${KUBEVIRT_MEMORY_SIZE:-16G}"
 export KUBEVIRT_TAG="${KUBEVIRT_TAG:-main}"
 
@@ -38,8 +35,8 @@ function kubevirt::install() {
 function kubevirt::up() {
   make cluster-up -C "${_base_dir}/_kubevirt" && make cluster-sync -C "${_base_dir}/_kubevirt"
 
-  echo "enabling feature gates to validate instance types and preferences"
-  ${_kubectl} patch kv/kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"commonInstancetypesDeployment": {"enabled": false},"developerConfiguration":{"featureGates": ["VMPersistentState"]},"vmStateStorageClass":"nfs-csi"}}}'
+  echo "disable common-instancetypes deployment"
+  ${_kubectl} patch kv/kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"commonInstancetypesDeployment": {"enabled": false}}}}'
 
   echo "waiting for kubevirt to become ready, this can take a few minutes..."
   ${_kubectl} -n kubevirt wait kv kubevirt --for condition=Available --timeout=15m

--- a/scripts/kubevirtci.sh
+++ b/scripts/kubevirtci.sh
@@ -16,9 +16,6 @@
 
 set -ex
 
-# Required by the VMPersistentState feature that currently needs a RWX FS mode SC
-# https://kubevirt.io/user-guide/virtual_machines/persistent_tpm_and_uefi_state/
-export KUBEVIRT_DEPLOY_NFS_CSI="${KUBEVIRT_DEPLOY_NFS_CSI:-true}"
 export KUBEVIRT_MEMORY_SIZE="${KUBEVIRT_MEMORY_SIZE:-16G}"
 export KUBEVIRTCI_TAG=${KUBEVIRTCI_TAG:-$(curl -sfL https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)}
 export KUBEVIRT_DEPLOY_CDI="true"
@@ -58,8 +55,8 @@ function kubevirtci::up() {
   echo "waiting for kubevirt to become ready, this can take a few minutes..."
   ${_kubectl} -n kubevirt wait kv kubevirt --for condition=Available --timeout=15m
 
-  echo "enabling feature gates to validate instance types and preferences"
-  ${_kubectl} patch kv/kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"commonInstancetypesDeployment": {"enabled": false},"developerConfiguration":{"featureGates": ["VMPersistentState"]},"vmStateStorageClass":"nfs-csi"}}}'
+  echo "disable common-instancetypes deployment"
+  ${_kubectl} patch kv/kubevirt -n kubevirt --type merge -p '{"spec":{"configuration":{"commonInstancetypesDeployment": {"enabled": false}}}}'
 
   echo "waiting for kubevirt to become ready, this can take a few minutes..."
   ${_kubectl} -n kubevirt wait kv kubevirt --for condition=Available --timeout=15m

